### ISSUE TITLE
Fix references to submodule in README build instructions

### ISSUE THREAD

**Issue by torarnv** [Vector35/workflow_objc#43]
The `.gitmodules` file is empty, so `git submodule update --init --recursive` doesn't do anything.

And building results in:

```
❯ cmake -S . -B build -GNinja
-- The C compiler identification is AppleClang 14.0.0.14000029
-- The CXX compiler identification is AppleClang 14.0.0.14000029
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at CMakeLists.txt:9 (add_subdirectory):
  add_subdirectory given source "Vendor/BinaryNinjaAPI" which is not an
  existing directory.


CMake Error at CMakeLists.txt:86 (bn_install_plugin):
  Unknown CMake command "bn_install_plugin".


-- Configuring incomplete, errors occurred!
See also "/Users/torarne/dev/binaryninja/workflow_objc/build/CMakeFiles/CMakeOutput.log".
See also "/Users/torarne/dev/binaryninja/workflow_objc/build/CMakeFiles/CMakeError.log".
```

I'm guessing there should be something in `Vendor`?

**Comment by jonpalmisc** [comment]
This is a leftover artifact from before this was a first-party plugin. Leaving this issue open to track correcting the README, but the absence of a submodule is intentional.

**Comment by torarnv** [comment]
Ah, I see. Where is the vendored dependency meant to come from, the one missing when I try to build?

```
CMake Error at CMakeLists.txt:9 (add_subdirectory):
  add_subdirectory given source "Vendor/BinaryNinjaAPI" which is not an
  existing directory.
```

**Comment by torarnv** [comment]
Added back submodule in https://github.com/Vector35/workflow_objc/pull/44, as it's required to build the plugin in its current state

**Comment by torarnv** [comment]
Thanks, I understand the reason for the removal now, and can built it locally with the current patch as well, so I'm good 👍🏻

**Comment by jonpalmisc** [comment]
No worries at all1 I'm actually going to re-open this since the current solution of having to patch locally isn't very user-friendly; I'd like to make the process easy for both users and employees.

**Comment by jonpalmisc** [comment]
This should be effectively fixed by #57.

**Comment by torarnv** [comment]
Thanks! :)
